webapp/src/shared/components/launcher/background/background.component.js

```diff
@@ -0,0 +1,14 @@
+import { Container, Sprite, Texture } from 'pixi.js';
+
+class Background extends Container {
```

Comment:
I would not want to use extends becuase it is very easy to overwrite the original prop which you don't know

Reply:
Good point, I'm going to refactor this.